src/Map.cpp

```diff
@@ -77,7 +77,7 @@ std::vector<std::string> Map::tokenize(std::string line) {
 void Map::exportData(std::string word, int numLines, int lineNum) {
     buffer += "(" + word + ", 1)\n";
     if ((buffer.size() >= bufferSize) || (lineNum == numLines - 1)) {
-        std::string tmpFile = boost::filesystem::path(inputFilename).stem().string() + "Output.txt";
+        std::string tmpFile = boost::filesystem::path(inputFilename).stem().string() + ".txt";
```

Comment:
@josephcranston `Map` is the only component writing to the temporary directory. Removed the `Output` addition which makes keep tracking of the file being processed a bit easier when dumping things into the output directory. Updated the test runner to match.

You might have had other, potentially future plans for this and would like it to stay - lmk and I can resolve the issue another way. This seemed like the easiest approach.

Reply:
@WesRoach No problem - I only added the "Output" as a placeholder.